### PR TITLE
LPS-132579 Fix SLAInfo Tests

### PR DESCRIPTION
**Pull request by jotascript**
Issue: [LPS-132579](https://issues.liferay.com/browse/LPS-132579)

### CODE REVIEW

**Comment by jotascript** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by ramonmsp — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/AppContext.es.js [hunk 1]
```diff
@@ -18,12 +18,15 @@ const AppContext = React.createContext();
 const AppContextProvider = ({children, ...props}) => {
 	const [reindexStatuses, setReindexStatuses] = useState([]);
 	const [title, setTitle] = useState(Liferay.Language.get('metrics'));
+	const [shouldCalculateMetrics, setShouldCalculateMetrics] = useState(false);
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/AppContext.es.js:21]: Hey @jotascript, Can you please rename to `setFetchDateModified` and `fetchDateModified`? Please propagate this changes to other places as well.

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by jotascript** [comment]
ci:forward

**Pull request closed without merging**